docs/ref/booking/notes.rst

```diff
@@ -0,0 +1,19 @@
+booking/notes
+--------------
+This is a call to your Checkfront account that pulls all the notes from all of your bookings within a certain time frame. By default this will pull all notes from all your bookings in the system. 
+
+.. http:post:: /api/3.0/booking/notes
+
+	:query string/timestamp date: Specify a date to show the notes from. Still testing the full extent of this. This can be a month and will show all notes from that month. This can be in the format "YYYY-MM-DDtHH:MM:SS-07:00" without quotes. It can be in the format "YYYY-MM-DD"
```

Comment:
"Specify a date to show the notes from" should be:
Filter the booking notes by a date